packages/backend-for-frontend/src/services/catalogService.ts

```diff
@@ -1786,5 +1786,27 @@ export function catalogServiceBuilder(
 
       return { id: descriptorId };
     },
+    async checkEServiceNameAvailability(
```

Comment:
minor, but more coherent with return value
```suggestion
    async isEServiceNameAvailable(
```